drop-boxes/register-convert-ms-vendor-format/README.md

```diff
@@ -0,0 +1,51 @@
+# Mass Spectrometry Dropbox
+
+## Expected data structure
+In every use case, the data structure needs to contain a top folder around the respective data in order to accommodate metadata files.
+
+The sample code found in the top folder can be of type `Q_TEST_SAMPLE` or `Q_MS_RUN`. In the former case, a new sample of type `Q_MS_RUN` is created and attached as child to the test sample.
+
+**Valid folder/file types**:
+- Thermo Fisher Raw file format
+- Waters Raw folder
+- Bruker .d folder
+
+**Incoming structure overview for standard case without additional metadata file:**
+```
+QABCD102A5_20201229145526_20201014_CO_0976StSi_R05_.raw
+|-- QABCD102A5_20201229145526_20201014_CO_0976StSi_R05_.raw
+|-- QABCD102A5_20201229145526_20201014_CO_0976StSi_R05_.raw.sha256sum
+```
+In this case, existing mass spectrometry metadata is expected to be already stored and the dataset will be attached.
+
+
+**Incoming structure overview for case for Immunomics data with metadata file:**
+```
+QABCD090B7
+|-- QABCD090B7
+|   |-- file1.raw
+|   |-- file2.raw
+|   |-- file3.raw
+|   `-- metadata.tsv
+|-- QABCD090B7.sha256sum
+`-- source_dropbox.txt
```

Comment:
what is about the metadata json as stated [here](https://github.com/qbicsoftware/etl-scripts/blob/b65dacc8abfa946198f6367e7fb2bcf1f4925a92/drop-boxes/register-convert-ms-vendor-format/etl_msconvert.py#L292) in the comment of one method?

Reply:
this is unused and seems to be from a really early stage of the script. nicely spotted!

Reply:
what shall we do about it?

Reply:
hm.. if its not used I would maybe also remove it out of the code at one point?
I thought json schemas are our golden way/we aim for them?

Reply:
it's really, really old...I will open an issue for it :)

Reply:
sorry for being so nitty 🙈  but I think even if it's old it helps to clean up the code and not lose track of the issue :)

Reply:
no no, I agree, I mean the golden way of json was not a thing back then, hence we went with something the labs wanted to use instead :)